### PR TITLE
add doc for enabling sso with oauthcredentials setting

### PR DESCRIPTION
**Pull request by lzc850612**
### Purpose
*What is the context of this pull request? Why is it being done?*

Add doc for enabling sso by using oauthcredentials settings.

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: docs/_docs/overview/whats-new/1.0/enable-sso-with-skills-using-oauthcredentials-setting.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+category: Overview
+subcategory: What's New
+language: 1_0_release
+title: Enable SSO with skills using OAuthCredentials setting
```
Comment (ryanisgrig) [inline docs/_docs/overview/whats-new/1.0/enable-sso-with-skills-using-oauthcredentials-setting.md]: Replace lowercase instances of skills to Skills

File: docs/_docs/overview/whats-new/1.0/enable-sso-with-skills-using-oauthcredentials-setting.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+## {{ page.title }}
+{:.no_toc}
+{{ page.description }}
+
+## Intro
+ 
+In the previous release (0.8) we added documentation about [enabling SSO for skills](https://microsoft.github.io/botframework-solutions/overview/whats-new/0.8-beta/achieve-SSO-among-skills/). In the 1.0 release this is officially included. When a Virtual Assistant has multiple skills added, in order to achieve SSO, one approach is all the skills to share the OAuth connection settings from one central Bot Channel Registration, preferrably from the Virtual Assistant. 
+
+1. Add OAuth connection setting that includes all the necessary scopes that are needed for all the skills
+   Go to Virtual Assistant's azure portal Settings tab, and click 'Add Setting' to add an oauth connection
```
Comment (ryanisgrig) [inline docs/_docs/overview/whats-new/1.0/enable-sso-with-skills-using-oauthcredentials-setting.md]: * Replace oauth with OAuth where used
* Replace azure portal with Azure portal

**Review by ryanisgrig — CHANGES_REQUESTED** [review]

**Review by lauren-mills — APPROVED** [review]

File: docs/_docs/overview/whats-new/1.0/enable-sso-with-skills-using-oauthcredentials-setting.md (hunk 1 above)
Comment (lzc850612) [reply docs/_docs/overview/whats-new/1.0/enable-sso-with-skills-using-oauthcredentials-setting.md]: done

File: docs/_docs/overview/whats-new/1.0/enable-sso-with-skills-using-oauthcredentials-setting.md (hunk 2 above)
Comment (lzc850612) [reply docs/_docs/overview/whats-new/1.0/enable-sso-with-skills-using-oauthcredentials-setting.md]: done

**Review by darrenj — APPROVED** [review]

**Review by ryanisgrig — APPROVED** [review]

**Pull request merged**